extensions/mssql/src/objectManagement/ui/objectManagementDialogBase.ts

```diff
@@ -72,44 +72,64 @@ export abstract class ObjectManagementDialogBase<ObjectInfoType extends ObjectMa
 		return this.options.isNewObject ? TelemetryActions.CreateObject : TelemetryActions.UpdateObject;
 	}
 
-	protected override async initialize(): Promise<void> {
-		await super.initialize();
-		this.dialogObject.registerOperation({
-			displayName: this.saveChangesTaskLabel,
-			description: '',
-			isCancelable: false,
-			operation: async (operation: azdata.BackgroundOperation): Promise<void> => {
-				try {
-					if (this.isDirty) {
-						const startTime = Date.now();
-						await this.saveChanges(this._contextId, this.objectInfo);
-						if (this.options.objectExplorerContext) {
-							if (this.options.isNewObject) {
-								await refreshNode(this.options.objectExplorerContext);
-							} else {
-								// For edit mode, the node context is the object itself, we need to refresh the parent node to reflect the changes.
-								await refreshParentNode(this.options.objectExplorerContext);
-							}
-						}
-
-						TelemetryReporter.sendTelemetryEvent(this.actionName, {
-							objectType: this.options.objectType
-						}, {
-							elapsedTimeMs: Date.now() - startTime
-						});
-						operation.updateStatus(azdata.TaskStatus.Succeeded);
+	protected get startsTaskSeparately(): boolean {
```

Comment:
Maybe something like `startTaskOnApply` instead (and reverse the boolean values) - "separately" isn't necessarily true (we could decide not to start a task at all for example) and isn't as clear to me how it affects this dialog.

Could use a doc comment too just to make it even clearer